### PR TITLE
Finish intro notebook

### PR DESCRIPTION
**Pull request by astrojuanlu**

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Pending push to https://github.com/ibis-project/kedro-ibis-tutorial ⚠️

File: 00 - Welcome.ipynb
Comment (deepyaman) [inline 00 - Welcome.ipynb:1]: Maybe just hit "Clear all outputs"?

**Review by deepyaman — APPROVED** [review]
One minor comment, but feel free to merge!

**Comment by astrojuanlu** [comment]
Closing in favour of https://github.com/ibis-project/kedro-ibis-tutorial/pull/1

**Pull request closed without merging**